src/dropbox/dropbox.methods.ts

```diff
@@ -0,0 +1,84 @@
+import * as grpc from 'grpc';
+import { ApprovalService } from "../approval/approval.service";
+import { StatusService } from "../status/status.service";
+import { TransferRepository } from "../transfer/transfer.repository";
+import { IApproverInfo, ICanApproveToUser } from "../approval/approvers.interface";
+import { TransferError, NotFoundError } from '../utils/errors/errors';
+import { ITransfer } from '../transfer/transfer.interface';
+import { ITransferInfo } from './info.interface';
+
+const approvalService: ApprovalService = new ApprovalService();
+const statusService: StatusService = new StatusService();
+
+export class DropboxMethods {
+
+    static async GetTransfersInfo(call: grpc.ServerUnaryCall<any>): Promise<ITransferInfo[]> {
+        const fileID = call.request.fileID;
+        const userID = call.request.userID;
+        const transfers: ITransfer[] = await TransferRepository.getMany({ fileID, userID });
+        if (!transfers.length) throw new NotFoundError();
+
+        const transfersInfo: Promise<ITransferInfo[]> = Promise.all(transfers.map(async (transfer: ITransfer) => {
+            const info = await statusService.getStatus(transfer._id || "");
```

Comment:
Why sending an empty string as a transfer ID to the status service?

Reply:
done.